third_party/CMakeLists.txt

```diff
@@ -21,3 +21,4 @@ add_subdirectory(serd)
 add_subdirectory(utf8proc)
 add_subdirectory(zstd)
 add_subdirectory(lz4)
+add_subdirectory(fmt)
```

Comment:
We use spdlog as our logging library. The spdlog library already includes the fmt library.
You can find it there: `third_party/spdlog/spdlog/fmt`
Instead of replicating the fmt library, i would suggest using the fmt bundled in the spdlog.

Reply:
Done